graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/SchemaGenerator.kt

```diff
@@ -47,7 +48,7 @@ import kotlin.reflect.full.createType
 */
 open class SchemaGenerator(internal val config: SchemaGeneratorConfig) : Closeable {
 
-    internal val additionalTypes: MutableSet<KType> = mutableSetOf()
+    internal val additionalTypes: MutableSet<AdditionalType> = mutableSetOf()
```

Comment:
instead of using wrapped type here why don't we store those separately in `additionalTypes` and `additionalInputTypes`?

Reply:
I could, we would just have to separate the logic out internally to call the methods twice for each set. So I thought it was easier to just have one and keep the looping logic unchanged

Reply:
Updated

Reply:
I added this logic in but resolving the types after generation gets more complicated because generating additional input types can lead to more `additionalTypes` being created which means we have to loop again.

Our one loop, one set system I think is better. The `AdditionalType` class is completely internal

Reply:
👍